media/css/firefox/os/firefox-os-new.less

```diff
@@ -0,0 +1,897 @@
+@import "../../sandstone/lib.less";
+
+* {
+    box-sizing: border-box;
+}
+#outer-wrapper {
+    background: #0c99d5 url(/media/img/firefox/os-new/raysoflight.png) 0 0 no-repeat;
+}
+#wrapper {
+    background-color: #0C99D5;
+    width: 100%;
+    padding-bottom: 0;
+}
+main {
+    margin: 0 auto;
+    text-align: center;
+    h1 {
+        background: transparent url(/media/img/firefox/os-new/fxos_wordmark420.png) 0 0 no-repeat;
+        margin: 0px auto 20px auto;
+        width: 440px;
+        height: 84px;
+
+        @media only screen and (max-width: 1366px) {
+            background-image: url(/media/img/firefox/os-new/fxos_wordmark420.png);
+            margin-bottom: 20px;
+            width: 420px;
+            height: 84px;
+        }
+
+        @media only screen and (max-width: 1024px) {
+            background-image: url(/media/img/firefox/os-new/fxos_wordmark320.png);
+            margin: 0px auto 10px auto;
+            width: 320px;
+            height: 60px;
+        }
+
+        @media only screen and (max-width: @breakMobile) {
+            background-image: url(/media/img/firefox/os-new/fxos_wordmark300.png);
+            margin: 0 auto 10px;
+            width: 300px;
+            height: 60px;
+        }
+        span {
+            .visually-hidden();
+        }
+    }
+    h2 {
+        .open-sans-light();
+        font-size: 3rem;
```

Comment:
Should this be an `em`? We do have a [font-size mixin](https://github.com/mozilla/bedrock/blob/master/media/css/sandstone/lib.less#L24) that uses rems and falls back to pixels for older browsers, if that may be of help.